puppet/environments/box/modules/profiles/files/box/cli/scripts/Command/Share.php

```diff
@@ -61,9 +69,32 @@ protected function _check()
         }
     }
 
+    protected function _removeVhostOverride()
+    {
+        $site_enabled = '/etc/apache2/sites-enabled/2-ngrok.conf';
+        $site_available = '/etc/apache2/sites-available/2-ngrok.conf';
+        $restart = false;
+
+        if (file_exists($site_enabled))
+        {
+            `sudo rm /etc/apache2/sites-enabled/2-ngrok.conf`;
```

Comment:
@yiendos I suggest to use the `a2dissite` command here to disable sites, that will remove the symlink using the suggested method.